### PR TITLE
Rename to ex_pcap. Format code. Update README

### PR DESCRIPTION
**Pull request by mickel8**
* cleanup repository
* setup CI
* add LICENSE
* rename to ex_pcap

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,15 @@
+   Copyright 2021 Software Mansion
+
+   Copyright 2021 Bryan Weber
```
Comment (bblaszkow06) [inline LICENSE:3]: Have you consulted how the License should look like for a fork?

File: README.md [hunk 2]
```diff
@@ -5,61 +5,22 @@ ExPcap
...
-This library parses pcap files, however it does not yet support most protocols
-that can be contained within a pcap file. The only supported protocols at the
-moment are:
+## Installation
 
-* Ethernet
-* IPv4
-* UDP
-* DNS
+The package can be installed by adding `ex_pcap` to your list of dependencies in `mix.exs`:
```
Comment (bblaszkow06) [inline README.md:13]: Don't you think this name (`ex_pcap`) is a bit similar to the original, that is published on hex? Is there any chance to take over the package as it is seems not maintained anymore?

File: lib/expcap.ex [hunk 3]
```diff
@@ -88,7 +102,7 @@ defmodule ExPcap do
   that contains a UDP packet that contains a DNS packet.
   """
   @spec parse_packet(nil, binary, [ExPcap.Packet.t()]) :: [ExPcap.Packet.t()]
-  def parse_packet(nil, _payload, acc) do
+  def parse_packet(nil, _, acc) do
```
Comment (bblaszkow06) [inline lib/expcap.ex]: This change is actually against what we used to do - naming all ignored variables

File: lib/expcap/binaries.ex [hunk 4]
```diff
@@ -214,8 +213,8 @@ defmodule ExPcap.Binaries do
       #<<1, 2, 3, 4>>
       "... redacted ..."
   """
-  @spec to_raw(binary) :: String.t
-  def to_raw(_b) do
+  @spec to_raw(binary) :: String.t()
+  def to_raw(_) do
```
Comment (bblaszkow06) [inline lib/expcap/binaries.ex]: name the variable

File: README.md (hunk 2 above)
Comment (mickel8) [reply README.md:13]: We have some other repositories with prefix `ex` e.g. `ex_libnice`, `ex_dtls`, `ex_libsrtp` so the idea was to make`expcap` similar to them. Even if we could take over the package I would rename it to `ex_pcap` :sweat_smile:

File: LICENSE (hunk 1 above)
Comment (mickel8) [reply LICENSE:3]: According to point 4 of the Apache License https://www.apache.org/licenses/LICENSE-2.0 and to https://softwareengineering.stackexchange.com/a/220079 I think that we did it properly. 
We included the original LICENSE with our copyrights, and we included original LICENSE with our copyrights and description of changes in the file we made some important changes.

**Review by bblaszkow06 — APPROVED** [review]

**Pull request merged**